src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -81,11 +68,31 @@ export const DefinitionList = ({
           <ul>
             {list.map((entity, i) => (
               <li key={i.toString()}>
-                <Link
-                  onClick={event => newSearchRequest(event, `\"${entity.name}\"`, 'author')}
-                  to={{ pathname: '/search', query: { q: `\"${entity.name}\"`, field: 'author' } }}
-                >{entity.name}, {entity.roles.join(', ')}
+                <Link to={{ pathname: '/search', query: linkToAuthor(entity) }}>
+                  {htmlEntities.decode(entity.name)}, {entity.roles.join(', ')}
                 </Link>
+                {(entity.birth_date_display || entity.death_date_display) && <span> (</span>}
```

Comment:
This seems too complicated to leave like this and not in its own function.

Reply:
changed